runmariadb

```diff
@@ -33,33 +40,41 @@ if [[ -f "$MARIADB_CERT_FILE" && "${RESTART_CONTAINER_CERTIFICATE_UPDATED}" == "
 fi
 
 if [ ! -d "${DATADIR}/mysql" ]; then
-    crudini --set "$MARIADB_CONF_FILE" mysqld max_connections "${MARIADB_MAX_CONNECTIONS}"
-    crudini --set "$MARIADB_CONF_FILE" mysqld max_heap_table_size 1M
-    crudini --set "$MARIADB_CONF_FILE" mysqld innodb_buffer_pool_size 5M
-    crudini --set "$MARIADB_CONF_FILE" mysqld innodb_log_buffer_size 512K
-    crudini --set "$MARIADB_CONF_FILE" mysqld user "${USER}"
+    cat > "$MARIADB_CONF_FILE" <<EOF
```

Comment:
For consistency, I'd like to see braces here, but up to you.

Reply:
There seems to be no consistency in this file...